src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,95 @@
+.cards {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  max-width: $card-wigth;
+
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    margin-right: 0;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-dark;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__code {
+    align-self: start;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    width: 100%;
+  }
+
+  &_review-counter {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    width: 100%;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-tag {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-dark;
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+    background: $color-blue;
+    border: 1px solid $color-blue;
+    text-align: center;
+    line-height: 40px;
+    border-radius: 5px;
+    text-transform: uppercase;
+    color: $color-white;
+    font-weight: 700;
+    font-size: 14px;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    color: $color-blue;
+    cursor: pointer;
+    background-color: $color-white;
+  }
+}
```

Comment:
Use nesting 
More details in [video](https://mate.academy/learn/css-advanced/sass#/video/925)